src/unstract/sdk/index.py

```diff
@@ -127,6 +127,7 @@ def index(
         reindex: bool = False,
         file_hash: Optional[str] = None,
         output_file_path: Optional[str] = None,
+        enable_highlight: bool = False,
```

Comment:
@johnyrahul I suggest renaming `enable_highlight` argument instead to something generic like `extract_with_metadata`

Reply:
@chandrasekharan-zipstack This argument is not something generic.  This is specifically for the text extraction logic alone.

meta_data will be always there as part of tool run, This flag/argument enables text extraction with whisper has only in case of llm_whisper and  updates whisper hash to metadata,.json